sites/public/pages/listings.tsx

```diff
@@ -57,17 +61,19 @@ const ListingsPage = () => {
     { value: "Foster City", label: "Foster City" },
   ]
 
-  function setPageAndFilterState(page: number, filters = filterState) {
+  function setQueryString(page: number, filters = filterState) {
     if (page != currentPage || filters != filterState) {
```

Comment:
I don't think we need this if check, since if we're calling `setQueryString()` then we already know we need to update the querystring

Reply:
Done